spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -20,13 +20,19 @@ mvn spring-boot:run
 ----
 
 Browse to `localhost:8080/log`. This will trigger the `ExampleController.log()` method,
-which uses a typical logger supporting Logback to write a line of text to the Stackdriver log. However,
-you will need to specify the trace ID in the `x-cloud-trace-context` header manually to see a trace ID
+which uses a typical logger supporting Logback to write a line of text to the Stackdriver log.
+However, you will need to specify the trace ID in the `x-cloud-trace-context` or `X-B3-TraceId` headers manually to see a trace ID
 label appear in Stackdriver:
 ----
 curl --header "x-cloud-trace-context: [Some-Trace-ID]" localhost:8080/log
 ----
 
+The `spring.cloud.gcp.logging.traceIdExtractor=xcloud_zipkin` setting in `application.properties`
```

Comment:
Also a nit, I've tried to use lower case-dash separated properties on all other docs. Would `spring.cloud.gcp.logging.trace-id-extractor` look better?
FWIW `spring.cloud.gcp.logging.trace-id-extractor` works as-is, no code changes are required.

Reply:
That's correct. It is called "Relaxed Binding" in Spring Boot: https://docs.spring.io/spring-boot/docs/1.5.9.RELEASE/reference/htmlsingle/#boot-features-external-config-relaxed-binding

So, even this will work:
```
SPRING_CLOUD_GCP_LOGGING_TRACE_ID_EXTRACTOR
```
Plus don't forget that there is a YAML style for properties: https://docs.spring.io/spring-boot/docs/1.5.9.RELEASE/reference/htmlsingle/#boot-features-external-config-yaml

Reply:
@ChengyuanZhao why didn't you change this?

Reply:
@joaoandremartins ,

why do you insist?
According the doc I mentioned above that doesn't matter and any possible value in our README is valid. I don't mind either way here though...

Reply:
It's how we're doing it on the rest of the documentation.
Let's have consistency, so our project has a polished look.